lib/OpenQA/WebAPI/Controller/API/V1/Job.pm

```diff
@@ -759,6 +759,25 @@ sub whoami {
 
 =over 4
 
+=item _test_suite_settings
+
+Retrieve test suite settings with setting extension from nested test suites
+using the keyword C<EXTENDS>.
+
+=back
+
+=cut
+
+sub _test_suite_settings {
```

Comment:
That this appears in the controller for the jobs API routes raises the question: Does this also work when using the isos post route? I suppose it should but I don't see this covered in the tests and there might be (duplicated) code which need to be adapted as well for this to work.

Reply:
holy moly, there is an ugly duplication in lib/OpenQA/Schema/Result/ScheduledProducts.pm … Let's see if I can fix that, but later